officeInsightIndex.js

```diff
@@ -7,6 +7,9 @@ var MEETING_ROOM_AVAILABILITY = '/rest/meetingRoom/availability/FIFTY_ONE';
 var MEETING_ROOM_AVAILABILITY_TIME = '?startTime=';
 var MEETING_ROOM_ASSISTANCE = '/rest/meetingRoom/assistance';
 var MEETING_ROOM_BOOK = '/rest/meetingRoom/book/';
+var GTFO_HOST = 'acheron.herokuapp.com';
```

Comment:
Donno if that'll work because our current Alexa integration involves one giant index file that we work with. We can probably upload a .zip with a more modularized approach, I think that should be a tech task though